k8s/simulation/network.yaml

```diff
@@ -1,7 +1,7 @@
-apiVersion: "simulator.io/v1"
+apiVersion: "keramik.3box.io/v1"
 kind: Network
 metadata:
-  name: network
-  namespace: ceramic
+  name: bob
```

Comment:
is the name/namespace set elsewhere?

Reply:
Yes, via kustimozation.yaml, this way the namespace is configurable for all related resources in one place.